api/repositories.py

```
@@ -29,13 +29,14 @@ class UserRepository(Repository):
     def __init__(self, mongo_collection, repository_collection):
         Repository.__init__(self, mongo_collection, repository_collection)
 
-    def update_user_account(self,user_id,name,password,house_name,house_location):
+    def update_user_account(self,user_id,name,password_hash,house_name,house_location):
         user = self.get_user_by_id(user_id)
         if user is None:
-            return None
+            return False
```

Why does this method return False here and a user object upon success?
I suggest you decide between a boolean return value (success/failure) or the up-to-date object upon succes and None upon failure.

---

If I return None here. Will the the value remain unchanged if the user is None?

---

if user is None, your endpoint should return a 404 error as the user cannot be found.

---

Yeah. That's what I did in api/main. But I think you said you there should also be a identifier here.